validation/shared/src/test/scala/hmda/validation/rules/lar/syntactical/S020Spec.scala

```diff
@@ -0,0 +1,16 @@
+package hmda.validation.rules.lar.syntactical
+
+import hmda.parser.fi.lar.LarGenerators
+import hmda.validation.dsl.Success
+import org.scalatest.{ MustMatchers, PropSpec }
+import org.scalatest.prop.PropertyChecks
+
+class S020Spec extends PropSpec with PropertyChecks with MustMatchers with LarGenerators {
+  property("Loan Application Register Agency Code must = 1,2,3,5,7,9") {
+    forAll(larGen) { lar =>
+      whenever(lar.id == 2) {
```

Comment:
True. It's usually necessary to use `whenever` to tell ScalaCheck what type of data to create, in this case it's a bit redundant. There might be some cases down the line where we only generate LARs that have a certain type of property